### ISSUE TITLE
ENH: Add automatic weighted loss for background class

### ISSUE THREAD

**Issue by NickleDave** [vocalpy/vak#607]
**Is your feature request related to a problem? Please describe.**
related to #606 

A nice-to-have would be ability to automatically tell a 2-D seg model to weight classes so that the background class has less impact on loss, this way dealing with class imbalance without needing to carefully curate the training set

sort of like in https://arxiv.org/pdf/1708.03211.pdf

**Describe the solution you'd like**

- :thinking: flag to any 2-d seg model that's something like `weight_bg_class=True`
- if True this triggers logic to compute a weights vector that then gets passed into the loss function when instantiating, as in https://androidkt.com/how-to-use-class-weight-in-crossentropyloss-for-an-imbalanced-dataset/
- my initial guess would be something like "1 / total number of classes besides background class"
